apps/chat/channels.py

```diff
@@ -393,9 +400,8 @@ def _transcribe_audio(self, audio: BytesIO) -> str:
             if speech_service.supports_transcription:
                 return speech_service.transcribe_audio(audio)
 
-    def _get_experiment_response(self, message: str) -> str:
-        experiment_bot = TopicBot(self.experiment_session)
-        answer = experiment_bot.process_input(message, attachments=self.message.attachments)
+    def _get_bot_response(self, message: str) -> str:
+        answer = self.bot.process_input(message, attachments=self.message.attachments)
```

Comment:
bot might be None here if a new session was created right?

Reply:
It wouldn't no, since we instantiate `self.bot` just after we called `self._ensure_sessions_exists()`. I think though it wouldn't hurt to just add a check here to make sure that the bot is always instantiated